core/runtime/module_factory.hpp

```diff
@@ -54,13 +55,17 @@ namespace kagome::runtime {
      * Compile `wasm` code to `path_compiled`.
      */
     virtual CompilationOutcome<void> compile(
-        std::filesystem::path path_compiled, BufferView wasm) const = 0;
+        std::filesystem::path path_compiled,
+        BufferView wasm,
+        const RuntimeContext::ContextParams &config) const = 0;
 
     /**
      * Load compiled code from `path_compiled`.
      */
     virtual CompilationOutcome<std::shared_ptr<Module>> loadCompiled(
-        std::filesystem::path path_compiled) const = 0;
+        std::filesystem::path path_compiled,
+        const std::optional<RuntimeContext::ContextParams> config =
```

Comment:
Why is config optional?

Reply:
Because here we have no config https://github.com/qdrvm/kagome/blob/2221-feature-request-handle-pvf-execution-parameters/core/parachain/pvf/kagome_pvf_worker.cpp#L354

Reply:
Then we should remove `optional` and pass config to worker
```diff
- using PvfWorkerInputCodePath = path
+ struct PvfWorkerInputCode { path, config }
```

Reply:
Why?

Reply:
WasmEdge interpreted mode doesn't use config in `compile`, so must receive config `loadCompiled`.
Pvf must respect `ExecutorParam::WasmExtBulkMemory`, so worker should receive config.